sentry-samples/sentry-samples-console/src/main/java/io/sentry/samples/console/Main.java

```diff
@@ -0,0 +1,114 @@
+package io.sentry.samples.console;
+
+import io.sentry.core.Breadcrumb;
+import io.sentry.core.EventProcessor;
+import io.sentry.core.Sentry;
+import io.sentry.core.SentryEvent;
+import io.sentry.core.SentryLevel;
+import io.sentry.core.protocol.Message;
+import io.sentry.core.protocol.User;
+import java.util.Collections;
+
+public class Main {
+
+  public static void main(String[] args) {
+    Sentry.init(
+        options -> {
```

Comment:
I'd also set:

```
options.setRelease("io.sentry.samples.console@3.0.0+1");
options.setSentryClientName("sentry-java/3.0.0");
```

while we don't read this automatically from the build conf/CI etc...so if people see this example that will land in master, they at least know that this is needed right now.

Reply:
I rather ppl didn't know this existed. We'll start receiving all sorts of interesting values on the server if ppl are pushed to add something here.